types/office-js-preview/index.d.ts

```diff
@@ -1536,9 +1536,15 @@ declare namespace Office {
          *
          * **Hosts**: Excel, OneNote, Outlook, PowerPoint, Word
          * 
-         * **Important**: In Outlook, this API is not supported if the add-in is loaded in an Outlook.com or Gmail mailbox. 
-         *
          * **Requirement set**: {@link https://docs.microsoft.com/javascript/api/requirement-sets/common/identity-api-requirement-sets | IdentityAPI 1.3}
+         * 
+         * **Important**: In Outlook, this API isn't supported if the add-in is loaded in an Outlook.com or Gmail mailbox.
+         * 
+         * **Note**: In an Outlook event-based activation add-in, this API is supported starting from Office on Windows version 2111 (build 14701.20000). 
+         * To retrieve an access token in older builds, use 
+         * {@link https://docs.microsoft.com/javascript/api/office-runtime/officeruntime.auth?view=common-js-preview#office-runtime-officeruntime-auth-getaccesstoken-member(1) | 
+         * OfficeRuntime.auth.getAccessToken} instead. For more information, see 
```

Comment:
I'm not sure if you can split the link like this - how will the asterisk be rendered or parsed?
```suggestion
         * {@link https://docs.microsoft.com/javascript/api/office-runtime/officeruntime.auth?view=common-js-preview#office-runtime-officeruntime-auth-getaccesstoken-member(1) | OfficeRuntime.auth.getAccessToken}
         * instead. For more information, see 
```

Reply:
I tested rendering this in the reference repo and verified that the link works.